doc/ref_model/chapters/chapter03.md

```diff
@@ -287,5 +287,20 @@ The terms Container Infrastructure Service Instance and Container Infrastructure
 <a name="3.8.2"></a>
 ### 3.8.2 Levels of accelerator consumption
 
-<a name="3.8.3"></a>
-### 3.8.3 Decoupling applications from infrastructure with Hardware Acceleration
+<a name="3.8.9"></a>
+### 3.8.9 Decoupling Applications from Infrastructure and Platform with Hardware Acceleration
+
+[Decoupling](https://github.com/cntt-n/CNTT/blob/master/doc/common/glossary.md#cloud-platform-abstraction-related-terminology) applications from hardware accelerator is normally accomplished using drivers that, if available, are preferred with standardised interfaces across vendors and their products, or if not available then through drivers specific to the vendor hardware device. Decoupling infrastructure software using standardised interfaces from hardware accelerator vendors is also preferred, and, if not available, is comparatively less of an issue by using one or few software infrastructure functions.
+
+Taking advantage of RM and RA environments with common capabilities, applications can be developed and deployed more rapidly, providing more service agility and easier operations. The extent to which this can be achieved will depend on levels of decoupling between application and infrastructure or platform underneath the application:
+
+#### 1. Infrastructure:
+- a) Application functionality or application control requires infrastructure components beyond RM profiles or infrastructure configuration changes beyond APIs specified by RA. Generally, such an application is tightly coupled with the infrastructure which results in an [Appliance deployment model](../../common/glossary.md#cloud-platform-abstraction-related-terminology).
+- b) Application control using APIs specified by RA finds nodes (already configured in support of the profiles) with the required infrastructure component(s), and in that node using APIs specified by RA configures infrastructure components that make application work. Example is an application that to achieve latency requirements needs certain hardware acceleration available in RM profile and is exposed through APIs specified by RA.
+- c) Application control using APIs specified by RA finds nodes (already configured in support of the profiles) with optional infrastructure component(s), and in these nodes using APIs specified by RA configures infrastructure component(s) that make application work better (like more performant) than without that infrastructure component. Example is an application that would have better TCO with certain acceleration adapter but can also work without it.
```

Comment:
Do you really mean TCO? Maybe replace with performance.

Reply:
> Do you really mean TCO? Maybe replace with performance.

@ulikleber : I changed it to "cost/performance" how it is called in sections "3.4.1 Hardware Acceleration Resources" #2158 and "3.8.5 Fixed Function Accelerators" #2167 . Thanks

Reply:
OK. Thx.